src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const costRentDay = 40;
+  const discountMin = 20;
+  const discountMax = 50;
+
+  if (days >= 7) {
+    return days * costRentDay - discountMax;
+  } else if (days >= 3) {
+    return days * costRentDay - discountMin;
+  }
```

Comment:
If you write `return` inside `if` block, using `else` after that doesn't make sense.
Structure should be like this:

```
if (condition1) {
  return x;
}

if (condition2) {
  return y
}

return z;
```